app/models/recipe.rb

```diff
@@ -4,4 +4,14 @@ class Recipe < ApplicationRecord
 
   validates :name, presence: true
   validates :description, presence: true
+
+  def count_food_recipes
+    recipe_foods.count
+  end
+
+  def sum_foods
+    sum = 0
+    Recipe.includes(:recipe_foods).find(id).recipe_foods.each { |item| sum += (item.food.price * item.quantity) }
+    sum
```

Comment:
Would it make sense to return a string that includes the $ symbol? This would match the format specified in the wireframe.